pkg/api/playlist.go

```diff
@@ -27,7 +27,9 @@ import (
 func (hs *HTTPServer) registerPlaylistAPI(apiRoute routing.RouteRegister) {
 	// Register the actual handlers
 	apiRoute.Group("/playlists", func(playlistRoute routing.RouteRegister) {
-		if hs.Features.IsEnabledGlobally(featuremgmt.FlagKubernetesPlaylists) {
+		// TODO: remove kubernetesPlaylists feature flag
+		if hs.shouldUseK8sAPI(playlistalpha1.PlaylistResourceInfo.GroupResource().String()) ||
```

Comment:
I wonder if `shouldUseK8sAPI` should/could be refactored to take into the account the cluster wide feature toggle too.

@leonorfmartins what do you think?

Reply:
Id say, we should have the same rule to enable the API in the BE and in the FE... so if we allow for cluster-wide feature toggles to control enabling the API here, we should also do the same in the FE?
I'm actually less inclined to do this for now, we can always add it later on if we need to?